packages/database/src/migrations/20210506052033-AddSamoaCovidPercentageVaccinatedMeasures-modifies-data.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getMeasureBuilderConfig = dataElementCode => ({
+  aggregations: [
+    {
+      type: 'FINAL_EACH_DAY',
+    },
+  ],
```

Comment:
I might be wrong, but from looking at `composePercentagePerOrgUnit` measure builder and `fetchComposedData`, I think this `aggregations` config will be ignored if you put it at the top level config like this (see `fetchComposedData()` in measureBuilders/helpers.js`. Instead, I think that you have to put this `aggregations` config in each of the sub measure builder config (numerator and denominator)